src/CanvasChart.js

```diff
@@ -246,44 +252,56 @@ export default function CanvasChart({
       const textX = taskX + AVATAR_SIZE + MARGIN;
       const textY = taskY - textHeight - MARGIN;
 
-      if (avatar?.image != null) {
-        drawAvatarCircle(
+      if (task.isMilestone) {
```

Comment:
Let's let the text fill the full width of the line rather than clip it within the (not really applicable "length")?

```diff
--git a/src/CanvasChart.js b/src/CanvasChart.js
index 296bac2..3871ac2 100644
--- a/src/CanvasChart.js
+++ b/src/CanvasChart.js
@@ -247,9 +247,11 @@ export default function CanvasChart({
       }
 
       // Render task owner avatars and labels.
-      const textWidth = taskWidth - AVATAR_SIZE - MARGIN;
-      const textHeight = AVATAR_SIZE;
       const textX = taskX + AVATAR_SIZE + MARGIN;
+      const textWidth = task.isMilestone
+        ? width - textX
+        : taskWidth - AVATAR_SIZE - MARGIN;
+      const textHeight = AVATAR_SIZE;
       const textY = taskY - textHeight - MARGIN;
 
       if (task.isMilestone) {
```

(Can't leave this comment on line 250 above because GitHub won't let me.)

Reply:
Not sure about this.... I think possibly I collided on a merge.